sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuFlatMapCoGroupsInPandasExec.scala

```diff
@@ -98,7 +98,9 @@ case class GpuFlatMapCoGroupsInPandasExec(
 
   override def requiredChildOrdering: Seq[Seq[SortOrder]] = {
     leftGroup
-      .map(SortOrder(_, Ascending)) :: rightGroup.map(SortOrder(_, Ascending)) :: Nil
+      .map(ShimLoader.getSparkShims.sortOrder(_, Ascending)) ::
+    rightGroup
+      .map(ShimLoader.getSparkShims.sortOrder(_, Ascending)) :: Nil
```

Comment:
nit: 2 space indent for line wrapping https://docs.scala-lang.org/style/indentation.html#line-wrapping

Reply:
Which one should it be ?
```
    leftGroup
      .map(SortOrder(_, Ascending)) :: rightGroup.map(SortOrder(_, Ascending)) :: Nil
      rightGroup
        .map(ShimLoader.getSparkShims.sortOrder(_, Ascending)) :: Nil
```
or 
```
    leftGroup
      .map(SortOrder(_, Ascending)) :: rightGroup.map(SortOrder(_, Ascending)) :: Nil
        rightGroup
          .map(ShimLoader.getSparkShims.sortOrder(_, Ascending)) :: Nil
```

Reply:
should be fine w/ just 
```
    leftGroup.map(ShimLoader.getSparkShims.sortOrder(_, Ascending)) ::
      rightGroup.map(ShimLoader.getSparkShims.sortOrder(_, Ascending)) :: Nil
```

Reply:
done